compiler/src/org.graalvm.compiler.replacements.aarch64/src/org/graalvm/compiler/replacements/aarch64/AArch64GraphBuilderPlugins.java

```diff
@@ -64,8 +64,8 @@ public static void register(Plugins plugins, BytecodeProvider bytecodeProvider,
         invocationPlugins.defer(new Runnable() {
             @Override
             public void run() {
-                registerIntegerLongPlugins(invocationPlugins, AArch64IntegerSubstitutions.class, JavaKind.Int, bytecodeProvider);
```

Comment:
This would make the `AArch64IntegerSubstitutions` and `AArch64LongSubstitutions` obsolete right?